modules/i18n/src/translations/en-US/portals/user.ts

```diff
@@ -368,6 +373,12 @@ export const user: userNS = {
                 userNotFound: "User not found"
             },
             totalInvitations: "Total Invitation(s)"
+        },
+        setPasswordModal: {
+            button: "Set Password",
+            header: "Set User Password",
+            message: "WARNING: Please note that after setting the password, the user will " +
```

Comment:
Since we already use a warning alert, we don't need to explicitly state that it is a warning in the error message. Also let's avoid `Please` in error messages.

```suggestion
            message: "After setting the password, the user will " +
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/fb3422572ea4918d0817c4d3802d2137fa0dd495